frontend/src/diagram/DiagramWebSocketContainer.tsx

```diff
@@ -613,7 +613,7 @@ export const DiagramWebSocketContainer = ({
 
   useEffect(() => {
     if (selectedObjectId && activeTool && contextualPalette) {
-      invokeTool(activeTool, contextualPalette.element.id, contextualPalette.startingPosition);
+      invokeTool(activeTool, contextualPalette.element, contextualPalette.startingPosition);
```

Comment:
The behavior of the frontend should stay the same, this change is related to an internal API of the backend